pkgs/applications/radio/qdmr/default.nix

```
@@ -0,0 +1,70 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeDesktopItem,
```

You forgot this:

```suggestion
  copyDesktopItems,
  makeDesktopItem,
```

P.S.: have you tried to test it before upload? Sometimes the local env is a bit heisenbuggy.

---

I just removed the desktop options in the last commit since cmake builds a desktop file automatically and also deploys it in the right place, tested it locally.